buildenv/jenkins/openjdk_tests

```diff
@@ -219,8 +219,9 @@ timestamps{
                         ['LABEL_ADDITION' : "${commonLabel}&&hw.arch.s390x.z15"]
                     ],
             'ppc64le_linux' : [
-                        ['LABEL_ADDITION' : "${commonLabel}&&hw.arch.ppc64le.p10"],
-                        ['LABEL_ADDITION' : "${commonLabel}&&hw.arch.ppc64le.p9"]
+                        // Temporarily exclude ubuntu 24 machines due to podman problem
```

Comment:
Could we add the issue number in the comment?

Reply:
Added issue number in the comment. Thanks.